docs/src/pages/components/transitions/SimpleGrow.js

```diff
@@ -25,53 +24,47 @@ const styles = theme => ({
     stroke: theme.palette.divider,
     strokeWidth: 1,
   },
-});
+}));
 
-class SimpleGrow extends React.Component {
-  state = {
-    checked: false,
-  };
+function SimpleGrow() {
+  const classes = useStyles();
+  const [checked, setChecked] = React.useState(false);
 
-  handleChange = () => {
-    this.setState(state => ({ checked: !state.checked }));
-  };
-
-  render() {
-    const { classes } = this.props;
-    const { checked } = this.state;
+  function handleChange() {
+    setChecked(prev => !prev);
+  }
 
-    const polygon = (
+  function Polygon() {
```

Comment:
To me, this should either be like the previous implementation or declared outside this component.